forms-flow-web/src/single-spa-index.js

```diff
@@ -22,11 +22,11 @@ if (typeof window.__REACT_DEVTOOLS_GLOBAL_HOOK__ === "object") {
   Formio.setBaseUrl(AppConfig.apiUrl);
   
   // Set custom formio elements - Code splitted
-  import("formsflow-formio-custom-elements/dist/customformio-ex").then(
-    (FormioCustomEx) => {
-      Components.setComponents(FormioCustomEx.components);
-    }
-  );
+  // import("formsflow-formio-custom-elements/dist/customformio-ex").then(
+  //   (FormioCustomEx) => {
+  //     Components.setComponents(FormioCustomEx.components);
+  //   }
+  // );
```

Comment:
This will be reverted back as part of https://aottech.atlassian.net/browse/FWF-3657